README_es.MD

```diff
@@ -0,0 +1,287 @@
+# cfn-lint [![CircleCI](https://img.shields.io/circleci/project/github/martysweet/cfn-lint.svg)](https://circleci.com/gh/martysweet/cfn-lint) [![npm version](https://badge.fury.io/js/cfn-lint.svg)](https://www.npmjs.com/package/cfn-lint) [![npm](https://img.shields.io/npm/dm/cfn-lint.svg)](https://www.npmjs.com/package/cfn-lint)
+
+[//]: # (Translation Section: A more friendly CloudFormation JSON and YAML Validator)
+#### Un validador para CloudFormation JSON and YAML más amigable.
+`cfn-lint` es una herramienta utilizada para mejorar tu ciclo de desarrollo en CloudFormation. Si estás aquí, probablemente estés 
+harto de esperar 10 minutos o más antes de que CloudFormation te dé un error debido a un error gramatical. Esta herramienta tiene como objetivo eliminar esa pérdida de tiempo. 
+La herramienta toma el modelo de CloudFormation, y resuelve todas las funciones intrínsecas definidas antes de verificar las _Propiedades_ de los _Recursos_ definidos.
+La herramienta puede usarse sobre la línea de comando usando `cfn-lint`, o puede usarse como un módulo dentro de otra aplicación de JavaScript.
+
+#* _Nota: actualmente, esta herramienta distingue entre mayúsculas y minúsculas en relación con los tipos AWS CloudFormation, por ejemplo aws::lambda::function != AWS::Lambda::Function._*
+
+[//]: # (Translation Section: Installation)
+## Instalación
+
+Se puede instalar con `npm`:
+
+```
+$ npm instalar -g cfn-lint
+```
+
+Si lo consigues `/usr/bin/env: 'node': No such file or directory (El archivo o directorio no existe)` Asegúrate de que tu sistema tenga NodeJs instalado. En los sistemas Debian, es posible que deba enlazar el node a nodejs debido a problemas de espacio con los nombres (`ln -s /usr/bin/nodejs /usr/bin/node`).
+
+[//]: # (Translation Section: How to use?)
+## ¿Cómo se usa?
+`cfn-lint validate my_template.yaml`
+
+`cfn-lint validate my_template.json`
+
+`cfn-lint validate my_template.yaml --parameters key="my value",key2=value2,key3=3`
+
+`cfn-lint validate my_template.yaml --parameters key="my value",key2=value2 --no-guess-parameters`
+
+`cfn-lint validate my_template.yaml --pseudo AWS::StackName="My-Stack"`
+
+`cfn-lint validate my_template.yaml --parameters key="my value" --pseudo AWS::Region=ap-northeast-1,AWS::AccountId=000000000000`
+
+`cfn-lint docs AWS::Lambda::Function` 
+
+`cfn-lint docs AWS::Lambda::Function.Code`
+
+`cfn-lint docs AWS::Lambda::Function.Code.S3Bucket`
+
+**Nota: El orden de `--parameters` y` --pseudo` actualmente importa. Esto debería abordarse en una nueva versión.**
+
+[//]: # (Translation Section: Example Output)
```

Comment:
This won't be the translated output of the program without additional work (which would be outside the scope of this PR).

Reply:
Should I delete this part?

Reply:
Nope, just restore the English version of the program output.